public/app/plugins/datasource/influxdb/influxQLMetadataQuery.ts

```diff
@@ -0,0 +1,61 @@
+import { InfluxQueryTag } from './types';
+import InfluxDatasource from './datasource';
+import { InfluxQueryBuilder } from './query_builder';
+
+const runExploreQuery = async (
+  type: string,
+  withKey: string | undefined,
+  target: { measurement: string | undefined; tags: InfluxQueryTag[]; policy: string | undefined },
+  datasource: InfluxDatasource
+): Promise<Array<{ text: string }>> => {
+  const builder = new InfluxQueryBuilder(target, datasource.database);
+  const q = builder.buildExploreQuery(type, withKey);
+  return await datasource.metricFindQuery(q);
+};
```

Comment:
```suggestion
const runExploreQuery = (
  type: string,
  withKey: string | undefined,
  target: { measurement: string | undefined; tags: InfluxQueryTag[]; policy: string | undefined },
  datasource: InfluxDatasource
): Promise<Array<{ text: string }>> => {
  const builder = new InfluxQueryBuilder(target, datasource.database);
  const q = builder.buildExploreQuery(type, withKey);
  return datasource.metricFindQuery(q);
};
```

I don't think you need this to wrap this in a promise